bundles/net.tourbook.device.garmin.fit/src/net/tourbook/device/garmin/fit/FitData.java

```diff
@@ -73,7 +74,8 @@
    private String                  _sessionIndex;
    private ZonedDateTime           _sessionStartTime;
 
-   private String                  _sportName;
+   private String                  _sportName = "";
+   private String                  _profileName = "";
```

Comment:
Do you also want UI.EMPTY_STRING to be used for string comparisons? For example, in the following:
`if (!_profileName.equals("")) {`
becomes
`if (!_profileName.equals(UI.EMPTY_STRING)) {`

Reply:
>Do you also want UI.EMPTY_STRING to be used for string comparisons? 

Yes, but instead of

`if (!_profileName.equals(UI.EMPTY_STRING)) {`

you should swap the first and the second parameter

`if (!UI.EMPTY_STRING.equals(_profileName)) {`

this is NULL save, I just started in my program code to use this way

Reply:
Got it, thanks